### ISSUE TITLE
Add a way to delete all private routes

### ISSUE THREAD

**Issue by zstadler** [IsraelHikingMap/Site#888]
## What I expect to happen
With the introduction of persistent map state, the map can easily becomes cluttered with private routes.
It would be good to add a way to delete all private routes at once

## What really happened
One has to remove each private routes separately.

## What I did that caused the issue - step by step
Add several private routes by recording, adding shared routes or planning, and then delete all of them.

## My environment
* Operating System and version: Mobile App
* Web Browser and version: all

## Thing I think the developers should know, images, links etc.
This functionality was available by clicking the Israel Hiking logo.

**Comment by HarelM** [comment]
In the past clicking the IHM logo opened a new tab and so the data was cleared, I don't think this is an intuitive/expected behavior now.
Probably needs a button at the end of the routes to allow to delete them all.
We should also consider removing this logo in the app as the logo exists in the shortcut and current running applications menu, I think it only takes up valuable space...

**Comment by HarelM** [comment]
In order not to clutter this sidebar the button is added only when there's more than 3 routes.

**Comment by valleyofdawn** [comment]
I like this option less because it adds an extra element to the interface and is hidden deep within it.

**Comment by valleyofdawn** [comment]
The delete all button only appears after you've manually added 2 or more routes.
When several routes are present due to a share or a recording, it's missing.

**Comment by HarelM** [comment]
@valleyofdawn Can you please share a screen shot or a share that causes this behavior?

**Comment by valleyofdawn** [comment]
So it's a question of number.
Why delete only after 4 routes are present?

**Comment by HarelM** [comment]
UX concern, I guess. To remove 3 route requires 6 clicks which sounds still reasonable. Below 3 routes I prefer not to clutter the UI.

**Comment by HarelM** [comment]
I don't see anything that needs to be done here. Closing. Please reopen if you have any additional requests.

**Comment by valleyofdawn** [comment]
Was there a keyboard shortcut added?
I suggest the "x" button should also be visible when no routes are selected. Because the new option 4 is relevant to all routes. When used this way there is no reason to show the various options, only to declare "This will remove all tracks and points, are you sure?" 

See for instance:
https://www.facebook.com/groups/994960670559126/2119140788141103/?comment_id=2119154534806395

**Comment by valleyofdawn** [comment]
Was there a keyboard shortcut added?

**Comment by HarelM** [comment]
This I issue was resolved. If you wish to add other functionality or have an issue with the current implementation please open a separate issue. Thanks ahead.
